infra/base-images/base-runner/Dockerfile

```diff
@@ -14,8 +14,15 @@
 #
 ################################################################################
 
+# Using multi-stage build to copy some LLVM binaries needed in the runner image.
+FROM gcr.io/oss-fuzz-base/base-clang AS base-clang
+
 FROM gcr.io/oss-fuzz-base/base-image
-MAINTAINER mike.aizatsky@gmail.com
+
+COPY --from=base-clang /usr/local/bin/llvm-cov /usr/local/bin/
```

Comment:
nit: Can add a comment that copying artifacts needed for code coverage. or move "FROM gcr.io/oss-fuzz-base/base-clang AS base-clang" closer to this so that your comment applies

Reply:
Added an extra comment.